modules/runner/src/integrationTest/java/org/apache/ignite/internal/distribution/zones/ItDistributionZonesFilterTest.java

```diff
@@ -126,14 +124,6 @@ void testFilteredDataNodesPropagatedToStable() throws Exception {
 
         TablePartitionId partId = new TablePartitionId(table.tableId(), 0);
 
-        assertValueInStorage(
```

Comment:
We somehow how to check that stable has been actually rewritten. Now we only check it at the end

Reply:
Added the check with appropriate todo